lib/puppet/functions/simplib/validate_port.rb

```diff
@@ -0,0 +1,39 @@
+# Validates whether or not the passed argument is a valid port
+# (i.e. between `1` - `65535`).
+Puppet::Functions.create_function(:'simplib::validate_port') do
```

Comment:
Has this been replaced by data type Simplib::Port and friends? https://github.com/simp/pupmod-simp-simplib/blob/master/types/port.pp